source/_integrations/vicare.markdown

```diff
@@ -96,16 +134,7 @@ The `climate.vicare_heating` {% term integration %} has the following mapping of
 | `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use `all` keyword instead of entity_id.
 | `hvac_mode` | no | New value of HVAC mode
```

Comment:
```suggestion
| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use `all` keyword instead of entity_id. |
| `hvac_mode` | no | New value of HVAC mode |
```